### ISSUE TITLE
BOSH deployed Influxdb emitter not sending metrics.

### ISSUE THREAD

**Issue by fewknow** [concourse/concourse#2381]
# Bug Report
The following can also be handy:

* Concourse version: 3.14.1
* Deployment type (BOSH/Docker/binary): BOSH
* Infrastructure/IaaS:  AWS 
* Browser (if applicable): Chrome
* Did this used to work?  No 

I have configured my bosh manifest for the atc to have influxdb metric emmiter according to (https://bosh.io/jobs/atc?source=github.com/concourse/concourse&version=3.14.1#p%3dvault) . 
 ```
 jobs:
  - release: concourse
    name: atc
    properties:
      log_level: debug

      token_signing_key: ((token_signing_key))
      external_url: ((external_url))
      
      vault:
        auth:
          backend: token
          client_token:<redacted>
        url: https://vault.mgmt.corvesta.net
        
      influxdb:
        database: telegraf
        username: <redacted>
        password: <redacted>
        url: http://10.90.34.27:8086 
```
 but it doesn't seem to be sending any stats to influxdb as I don't see any in the UI.  Is there a way I can check this or is there something else that needs to be done?

I do see it registered a `web-0` so I am assuming it saw this web server.  I have 1 web and 10 workers.

<img width="876" alt="screen shot 2018-07-16 at 12 27 31 pm" src="https://user-images.githubusercontent.com/22297806/42770983-b4f17212-88f3-11e8-873d-df575a102f1b.png">

**Comment by vito** [comment]
Can you check the `atc.stdout.log` logs on the `web` node?

```
bosh -e <env> -d <deployment> logs web
```

**Comment by fewknow** [comment]
```
{"timestamp":"1531850474.725229025","source":"atc","message":"atc.syncer.provider.patient_search_history-postgres:radar.scan-resource-type.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,54],"pipeline-scoped-name":"provider.patient_search_history-postgres:resource-type:slack-notification","session":"35.93.1.1.1582.2"}}
{"timestamp":"1531850474.725268364","source":"atc","message":"atc.syncer.provider.patient_search_history-postgres:radar.scan-resource-type.interval-runner.tick.lock.did-not-get-lock","log_level":0,"data":{"pipeline-scoped-name":"provider.patient_search_history-postgres:resource-type:slack-notification","resource-type":"slack-notification","session":"35.93.1.1.1582.1"}}
{"timestamp":"1531850474.763282061","source":"atc","message":"atc.syncer.practitioner-postgres:radar.scan-resource.interval-runner.tick.acquire.acquired","log_level":0,"data":{"id":[0,208],"pipeline-scoped-name":"practitioner-postgres:resource:resource-data.architecture","session":"35.61.11.1.6944.2"}}
{"timestamp":"1531850474.784105301","source":"atc","message":"atc.syncer.practitioner-postgres:radar.scan-resource.interval-runner.tick.found-created-container-in-db","log_level":0,"data":{"container":"4185719d-d552-4b93-6422-e6e03febf1cc","pipeline-scoped-name":"practitioner-postgres:resource:resource-data.architecture","session":"35.61.11.1.6944"}}
{"timestamp":"1531850474.799703360","source":"atc","message":"atc.syncer.practitioner-postgres:radar.scan-resource.interval-runner.tick.checking","log_level":0,"data":{"from":{"ref":"94ec7687534c63975b2eed07b5ea368908aad9dd"},"pipeline-scoped-name":"practitioner-postgres:resource:resource-data.architecture","session":"35.61.11.1.6944"}}
{"timestamp":"1531850474.846230030","source":"atc","message":"atc.syncer.practitioner.he.proxy:radar.scan-resource-type.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,54],"pipeline-scoped-name":"practitioner.he.proxy:resource-type:slack-notification","session":"35.23.1.1.6907.2"}}
{"timestamp":"1531850474.846277237","source":"atc","message":"atc.syncer.practitioner.he.proxy:radar.scan-resource-type.interval-runner.tick.lock.did-not-get-lock","log_level":0,"data":{"pipeline-scoped-name":"practitioner.he.proxy:resource-type:slack-notification","resource-type":"slack-notification","session":"35.23.1.1.6907.1"}}
{"timestamp":"1531850474.878375053","source":"atc","message":"atc.syncer.benefits.he.proxy:radar.scan-resource.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,12],"pipeline-scoped-name":"benefits.he.proxy:resource:Terraform Deploy","session":"35.21.9.1.6812.2"}}
{"timestamp":"1531850474.878417969","source":"atc","message":"atc.syncer.benefits.he.proxy:radar.scan-resource.interval-runner.tick.lock.did-not-get-lock","log_level":0,"data":{"pipeline-scoped-name":"benefits.he.proxy:resource:Terraform Deploy","resource":"Terraform Deploy","session":"35.21.9.1.6812.1"}}
{"timestamp":"1531850474.885533810","source":"atc","message":"atc.syncer.pricing.rating_engine-postgres:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,62],"pipeline":"pricing.rating_engine-postgres","session":"35.92.9521.1.1"}}
{"timestamp":"1531850474.906234026","source":"atc","message":"atc.syncer.correspondence.proxy:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,16],"pipeline":"correspondence.proxy","session":"35.16.41662.1.1"}}
{"timestamp":"1531850474.920410156","source":"atc","message":"atc.skymarshal.user.not-authorized","log_level":0,"data":{"session":"5.303498"}}
{"timestamp":"1531850474.933600187","source":"atc","message":"atc.syncer.broker-postgres:radar.scan-resource.interval-runner.tick.acquire.acquired","log_level":0,"data":{"id":[0,475],"pipeline-scoped-name":"broker-postgres:resource:pullrequest","session":"35.67.9.1.7010.2"}}
{"timestamp":"1531850474.954080582","source":"atc","message":"atc.syncer.broker-postgres:radar.scan-resource.interval-runner.tick.found-created-container-in-db","log_level":0,"data":{"container":"8e3d728f-41a7-459a-7698-be9be9925d32","pipeline-scoped-name":"broker-postgres:resource:pullrequest","session":"35.67.9.1.7010"}}
{"timestamp":"1531850474.970597267","source":"atc","message":"atc.syncer.core.code-lists.service:radar.scan-resource.interval-runner.tick.check-failed","log_level":1,"data":{"exit-status":128,"pipeline-scoped-name":"core.code-lists.service:resource:TRAIN","session":"35.83.12.1.2887"}}
{"timestamp":"1531850474.973143101","source":"atc","message":"atc.syncer.core.code-lists.service:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,695],"pipeline-scoped-name":"core.code-lists.service:resource:TRAIN","session":"35.83.12.1.2887.4"}}
{"timestamp":"1531850474.980865717","source":"atc","message":"atc.syncer.pricing.rating_engine-postgres:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,62],"pipeline":"pricing.rating_engine-postgres","session":"35.92.9521.1.2"}}
{"timestamp":"1531850474.983112335","source":"atc","message":"atc.syncer.broker-postgres:radar.scan-resource.interval-runner.tick.checking","log_level":0,"data":{"from":{"branch":"develop","commit":"3f6ff2f88bd4","id":"3"},"pipeline-scoped-name":"broker-postgres:resource:pullrequest","session":"35.67.9.1.7010"}}
{"timestamp":"1531850475.046479464","source":"atc","message":"atc.syncer.correspondence.proxy:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,16],"pipeline":"correspondence.proxy","session":"35.16.41662.1.2"}}
{"timestamp":"1531850475.061058521","source":"atc","message":"atc.syncer.benefit.portal:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,39],"pipeline":"benefit.portal","session":"35.48.41784.1.1"}}
{"timestamp":"1531850475.062530041","source":"atc","message":"atc.syncer.group-account.he.proxy:radar.scan-resource.interval-runner.tick.no-new-versions","log_level":0,"data":{"pipeline-scoped-name":"group-account.he.proxy:resource:MASTER","session":"35.25.4.1.6929"}}
{"timestamp":"1531850475.065196276","source":"atc","message":"atc.syncer.group-account.he.proxy:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,261],"pipeline-scoped-name":"group-account.he.proxy:resource:MASTER","session":"35.25.4.1.6929.4"}}
{"timestamp":"1531850475.100751877","source":"atc","message":"atc.syncer.member.portal:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,47],"pipeline":"member.portal","session":"35.54.41799.1.1"}}
{"timestamp":"1531850475.155250549","source":"atc","message":"atc.syncer.benefit.portal:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,39],"pipeline":"benefit.portal","session":"35.48.41784.1.2"}}
{"timestamp":"1531850475.172528028","source":"atc","message":"atc.syncer.broker-postgres:radar.scan-resource.interval-runner.tick.check-failed","log_level":1,"data":{"exit-status":22,"pipeline-scoped-name":"broker-postgres:resource:pullrequest","session":"35.67.9.1.7010"}}
{"timestamp":"1531850475.176556110","source":"atc","message":"atc.syncer.broker-postgres:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,475],"pipeline-scoped-name":"broker-postgres:resource:pullrequest","session":"35.67.9.1.7010.4"}}
{"timestamp":"1531850475.182616472","source":"atc","message":"atc.syncer.practitioner.proxy:radar.scan-resource.interval-runner.tick.no-new-versions","log_level":0,"data":{"pipeline-scoped-name":"practitioner.proxy:resource:resource-devops.ci","session":"35.17.5.1.2890"}}
{"timestamp":"1531850475.183292150","source":"atc","message":"atc.syncer.member.portal:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,47],"pipeline":"member.portal","session":"35.54.41799.1.2"}}
{"timestamp":"1531850475.184342861","source":"atc","message":"atc.syncer.practitioner.proxy:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,333],"pipeline-scoped-name":"practitioner.proxy:resource:resource-devops.ci","session":"35.17.5.1.2890.4"}}
{"timestamp":"1531850475.241175652","source":"atc","message":"atc.syncer.provider-patient-search-history.service:radar.scan-resource-type.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,84],"pipeline-scoped-name":"provider-patient-search-history.service:resource-type:git-bitbucket-pr","session":"35.43.1.1.6893.2"}}
{"timestamp":"1531850475.241225719","source":"atc","message":"atc.syncer.provider-patient-search-history.service:radar.scan-resource-type.interval-runner.tick.lock.did-not-get-lock","log_level":0,"data":{"pipeline-scoped-name":"provider-patient-search-history.service:resource-type:git-bitbucket-pr","resource-type":"git-bitbucket-pr","session":"35.43.1.1.6893.1"}}
{"timestamp":"1531850475.291972160","source":"atc","message":"atc.syncer.provider.portal:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,60],"pipeline":"provider.portal","session":"35.88.41775.1.1"}}
{"timestamp":"1531850475.293262243","source":"atc","message":"atc.syncer.broker.portal:radar.scan-resource.interval-runner.tick.check-failed","log_level":1,"data":{"exit-status":128,"pipeline-scoped-name":"broker.portal:resource:STAGE","session":"35.49.9.1.6939"}}
{"timestamp":"1531850475.294193506","source":"atc","message":"atc.syncer.broker.portal:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,376],"pipeline-scoped-name":"broker.portal:resource:STAGE","session":"35.49.9.1.6939.4"}}
{"timestamp":"1531850475.369834900","source":"atc","message":"atc.syncer.provider.portal:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,60],"pipeline":"provider.portal","session":"35.88.41775.1.2"}}
{"timestamp":"1531850475.425473213","source":"atc","message":"atc.syncer.accounting-general-ledger-postgres:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,34],"pipeline":"accounting-general-ledger-postgres","session":"35.64.41792.1.1"}}
{"timestamp":"1531850475.452479601","source":"atc","message":"atc.syncer.member.portal:radar.scan-resource.interval-runner.tick.check-failed","log_level":1,"data":{"exit-status":128,"pipeline-scoped-name":"member.portal:resource:STAGE","session":"35.53.8.1.6939"}}
{"timestamp":"1531850475.453573704","source":"atc","message":"atc.syncer.member.portal:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,560],"pipeline-scoped-name":"member.portal:resource:STAGE","session":"35.53.8.1.6939.4"}}
{"timestamp":"1531850475.456124067","source":"atc","message":"atc.syncer.helloworld.proxy:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,50],"pipeline":"helloworld.proxy","session":"35.74.41687.1.1"}}
{"timestamp":"1531850475.508599520","source":"atc","message":"atc.syncer.accounting-general-ledger-postgres:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,34],"pipeline":"accounting-general-ledger-postgres","session":"35.64.41792.1.2"}}
{"timestamp":"1531850475.549485922","source":"atc","message":"atc.syncer.helloworld.proxy:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,50],"pipeline":"helloworld.proxy","session":"35.74.41687.1.2"}}
{"timestamp":"1531850475.569174767","source":"atc","message":"atc.syncer.correspondence.service:radar.scan-resource.interval-runner.tick.acquire.acquired","log_level":0,"data":{"id":[0,450],"pipeline-scoped-name":"correspondence.service:resource:TRAIN","session":"35.39.3.1.6940.2"}}
{"timestamp":"1531850475.589733601","source":"atc","message":"atc.syncer.correspondence.service:radar.scan-resource.interval-runner.tick.found-created-container-in-db","log_level":0,"data":{"container":"a379c971-e826-4525-7581-a9034c801b93","pipeline-scoped-name":"correspondence.service:resource:TRAIN","session":"35.39.3.1.6940"}}
{"timestamp":"1531850475.595080614","source":"atc","message":"atc.syncer.practitioner-postgres:radar.scan-resource.interval-runner.tick.no-new-versions","log_level":0,"data":{"pipeline-scoped-name":"practitioner-postgres:resource:resource-data.architecture","session":"35.61.11.1.6944"}}
{"timestamp":"1531850475.596180677","source":"atc","message":"atc.syncer.practitioner-postgres:radar.scan-resource.interval-runner.tick.release.released","log_level":0,"data":{"id":[0,208],"pipeline-scoped-name":"practitioner-postgres:resource:resource-data.architecture","session":"35.61.11.1.6944.4"}}
{"timestamp":"1531850475.608839750","source":"atc","message":"atc.syncer.correspondence.service:radar.scan-resource.interval-runner.tick.checking","log_level":0,"data":{"from":null,"pipeline-scoped-name":"correspondence.service:resource:TRAIN","session":"35.39.3.1.6940"}}
{"timestamp":"1531850475.643456459","source":"atc","message":"atc.syncer.core-email-service:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,37],"pipeline":"core-email-service","session":"35.38.41680.1.1"}}
{"timestamp":"1531850475.656860590","source":"atc","message":"atc.skymarshal.user.not-authorized","log_level":0,"data":{"session":"5.303499"}}
{"timestamp":"1531850475.730787039","source":"atc","message":"atc.syncer.core-email-service:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,37],"pipeline":"core-email-service","session":"35.38.41680.1.2"}}
{"timestamp":"1531850475.795403481","source":"atc","message":"atc.syncer.accounting-banking-service:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,41],"pipeline":"accounting-banking-service","session":"35.34.41671.1.1"}}
{"timestamp":"1531850475.824341297","source":"atc","message":"atc.syncer.member.he.proxy:radar.scan-resource.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,68],"pipeline-scoped-name":"member.he.proxy:resource:resource-build-java","session":"35.71.7.1.6769.2"}}
{"timestamp":"1531850475.824400902","source":"atc","message":"atc.syncer.member.he.proxy:radar.scan-resource.interval-runner.tick.lock.did-not-get-lock","log_level":0,"data":{"pipeline-scoped-name":"member.he.proxy:resource:resource-build-java","resource":"resource-build-java","session":"35.71.7.1.6769.1"}}
{"timestamp":"1531850475.864695787","source":"atc","message":"atc.syncer.broker-service:radar.scan-resource.interval-runner.tick.acquire.acquired","log_level":0,"data":{"id":[0,487],"pipeline-scoped-name":"broker-service:resource:QA","session":"35.35.14.1.6921.2"}}
{"timestamp":"1531850475.896290064","source":"atc","message":"atc.syncer.accounting-banking-service:scheduler.tick.lock.release.released","log_level":0,"data":{"id":[2,41],"pipeline":"accounting-banking-service","session":"35.34.41671.1.2"}}
{"timestamp":"1531850475.898431778","source":"atc","message":"atc.syncer.member.portal:radar.scan-resource.interval-runner.tick.acquire.acquired","log_level":0,"data":{"id":[0,461],"pipeline-scoped-name":"member.portal:resource:resource-devops.ci","session":"35.53.3.1.6938.2"}}
{"timestamp":"1531850475.909712315","source":"atc","message":"atc.syncer.broker-service:radar.scan-resource.interval-runner.tick.found-created-container-in-db","log_level":0,"data":{"container":"ccab1821-da55-4bad-75ad-90acef89e44b","pipeline-scoped-name":"broker-service:resource:QA","session":"35.35.14.1.6921"}}
{"timestamp":"1531850475.935715437","source":"atc","message":"atc.syncer.member.portal:radar.scan-resource.interval-runner.tick.found-created-container-in-db","log_level":0,"data":{"container":"516f505f-1399-4df8-6265-3e23885be2c7","pipeline-scoped-name":"member.portal:resource:resource-devops.ci","session":"35.53.3.1.6938"}}
{"timestamp":"1531850475.937198162","source":"atc","message":"atc.syncer.broker-service:radar.scan-resource.interval-runner.tick.checking","log_level":0,"data":{"from":{"ref":"5b0bf674449b04c12d328c895ec6cf3cff73e134"},"pipeline-scoped-name":"broker-service:resource:QA","session":"35.35.14.1.6921"}}
{"timestamp":"1531850475.980656147","source":"atc","message":"atc.syncer.member.portal:radar.scan-resource.interval-runner.tick.checking","log_level":0,"data":{"from":{"ref":"391e14aa09af4d7fa0cf60a3273cc4513789ce86"},"pipeline-scoped-name":"member.portal:resource:resource-devops.ci","session":"35.53.3.1.6938"}}
{"timestamp":"1531850476.000424385","source":"atc","message":"atc.syncer.broker.portal:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,32],"pipeline":"broker.portal","session":"35.50.41792.1.1"}}
{"timestamp":"1531850476.012671947","source":"atc","message":"atc.syncer.core.code-lists.service:scheduler.tick.lock.acquire.acquired","log_level":0,"data":{"id":[2,58],"pipeline":"core.code-lists.service","session":"35.84.41634.1.1"}}
{"timestamp":"1531850476.054731607","source":"atc","message":"atc.skymarshal.user.not-authorized","log_level":0,"data":{"session":"5.303500"}}
{"timestamp":"1531850476.076522827","source":"atc","message":"atc.syncer.core.code-lists.service:radar.scan-resource-type.interval-runner.tick.acquire.not-acquired-already-held-locally","log_level":0,"data":{"id":[0,84],"pipeline-scoped-name":"core.code-lists.service:resource-type:git-bitbucket-pr","session":"35.83.2.1.6906.2"}}
```
Here is just a small part of the file.    Looks like it would be data that would go to influxdb for metrics as it has the timestamp and info on activities being performed.

Is there any other logs files that I could inspect to get an indication if the data isn't making its way to influxdb?   

Thanks

**Comment by tleong000** [comment]
I'm also seeing the same thing.  We are running 3.14.1 on vSphere, and we are not receiving metrics in influx.  Connectivity is confirmed.  Config is confirmed.  I also don't know what logs to share.

**Comment by fewknow** [comment]
@vito giving this a bump.  Is there something else we can do to help diagnose?   Is this something that would have been addressed in the v4.0.0 release?  Thanks for your time.

**Comment by fewknow** [comment]
Not sure if this was ever fixed in a later release.  I plan to upgrade to v4+ and use datadog as we have an account.   I would hope this is available for those that don't have datadog. 

THanks
